src/styleguide/_patterns/04-templates/00-homepage.mustache

```diff
@@ -4,5 +4,11 @@
   {{#sections}}
   {{>organisms-section}}
   {{/sections}}
+  <section>
+    <p class="grav-u-text-centered">Here are some of the clients we have worked with:</p>
+    <div class="grav-o-container-background-banner">
```

Comment:
there is a bigger piece of work that I think it would be worth to address:

<img width="623" alt="screen shot 2018-04-16 at 16 24 59" src="https://user-images.githubusercontent.com/249718/38819708-0a990588-4194-11e8-9ded-2bd0734357d7.png">

By looking at the above screen, the previous paragraph seems to be belonging to the previous section, while it's not. You're basically using colour to group sections, so I'd suggest a couple of changes here which should also simplify the whole thing:

1. remove the wrapping div for the `list-image-links`, this is not needed anymore as it was providing the background
2. change the `grav-o-container-background-banner` to be something like a style modifier for the section that would set the background to it
3. while doing the above you might need to adjust margins and paddings to better separate the current section to the previous one
4. switching back into using the `organism-section` will then be quite straightforward, so if you do so, remember to update the examples for the organism and its description/documentation perhaps.